File: src/frc/robot/classes/PID.java
package frc.robot.classes;
public class PID {
    public double p;
    public double i;
    public double d;
    public double f;
    public PID(double _p, double _i, double _d) {
        p = _p;
        i = _i;
        d = _d;
        f = 0.0;
    }
    public PID(double _p, double _i, double _d, double _f) {
        p = _p;
        i = _i;
        d = _d;
        f = _f;
    }
}

File: src/main/kotlin/frc/robot/classes/PID.kt
package frc.robot.classes;
open class PID {
    val p: Double;
    val i: Double;
    val d: Double;
    constructor(_p: Double, _i: Double, _d: Double) {
        this.p = _p;
        this.i = _i;
        this.d = _d;
    }
}